chatgpt/utils/validator.py

```diff
@@ -185,7 +193,7 @@ def validate_sample(data: List[ChatGPTData]) -> bool | dict[str, float | bool]:
 
     return {
         'is_valid': avg_score >= threshold_score,
-        'score': avg_score / 100
+        'score': avg_score
```

Comment:
We normalized the score from 0-1 since other scores (like validator scores) were in that range too.